tests/test_shared_id.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2023 Google LLC.
```

Comment:
If you haven't yet, please run `npm run yapf` to ensure consistent formatting.

We should continue to do so manually until some CI/pre-commit PR like https://github.com/GoogleChromeLabs/chromium-bidi/pull/373 is merged

Reply:
run, but it looks terrible )

Reply:
If it looks terrible, we could change the coding style convention (see `.style.yapf`) to one of your liking. Or we could change to another formatter like `black`.